utils.sh

```diff
@@ -185,3 +185,14 @@ function bmo_config_map {
     
     cp ocp/deploy/metal3-config.yaml assets/generated/99_metal3-config.yaml
 }
+
+function image_mirror_config {
+    if [ ! -z "${MIRROR_IMAGES}" ]; then
+        cat << EOF
+imageContentSources:
+- mirrors:
+    - 192.168.111.1:5000/localimages/local-release-image
+  source: ${OPENSHIFT_RELEASE_IMAGE}
```

Comment:
The installer reports to me: 

```
level=fatal msg="failed to fetch Master Machines: failed to load asset \"Install Config\": invalid \"install-config.yaml\" file: imageContentSources[0].source: Invalid value: \"registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-204427\": must be repository--not reference"
```

The rendered YAML looks like:

```
- mirrors:
    - 192.168.111.1:5000/localimages/local-release-image
  source: registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-204427
```

Reading https://github.com/openshift/installer/blob/master/docs/dev/alternative_release_image_sources.md#installconfig, I'm not really sure what it's supposed to be - maybe just `registry.svc.ci.openshift.org/ocp/release`?

Reply:
Running with `--dry-run` option prints next message:
```
info: Dry run complete

Success
Update image:  192.168.123.1:5000/localimages/local-release-image:4.3.0-0.ci-2019-11-18-165709
Mirror prefix: 192.168.123.1:5000/localimages/local-release-image

To use the new mirrored repository to install, add the following section to the install-config.yaml:

imageContentSources:
- mirrors:
  - 192.168.123.1:5000/localimages/local-release-image
  source: registry.svc.ci.openshift.org/ocp/4.3-2019-11-18-165709
- mirrors:
  - 192.168.123.1:5000/localimages/local-release-image
  source: registry.svc.ci.openshift.org/ocp/release
```

Reply:
I just had a successful deployment with

```yaml
imageContentSources:
- mirrors:
    - 192.168.111.1:5000/localimages/local-release-image
  source: registry.svc.ci.openshift.org/ocp/release
- mirrors:
    - 192.168.111.1:5000/localimages/local-release-image
  source: registry.svc.ci.openshift.org/ocp/4.3-2019-11-18-165709
```